gdb-multi-arch/conda_build_config.yaml

```diff
@@ -313,7 +313,7 @@ dcap:
 eclib:
   - '20230424'
 elfutils:
-  - 0.189
+  - 0.189.memfault0_libdebuginfod
```

Comment:
Not sure if this is the correct way to ensure that the `..._libdebuginfod` version gets picked up. It seems to work though?

Reply:
yup, this should work!